docs/workforce-management-framework/topics/con-datasync-cloud-module.adoc

```diff
@@ -0,0 +1,42 @@
+= {WFM-RC-NameLong} Datasync Module
+
+As with all synchronisation tasks Datasync Module has cloud and client sides. This is a implementation of 
+the Feedhenry Sync Framework for advanced usage see the 
```

Comment:
From a downstream POV, "Feedhenry" should be avoided. Should FeedHenry be a variable?

Reply:
"Sync Framework. For advanced usage,  see the "

Reply:
agree